src/WebJobs.Script.Grpc/azure-functions-language-worker-protobuf/src/proto/FunctionRpc.proto

```diff
@@ -326,7 +326,7 @@ message RpcFunctionMetadata {
   // Properties for function metadata
   // They're usually specific to a worker and largely passed along to the controller API for use
   // outside the host
-   map<string,string> Properties = 16;
+   map<string,string> properties = 16;
```

Comment:
Any breaking change concern here? I am not sure what release state this is already in or gRPC specifics.

Reply:
I actually pulled this change from protobuf repo. It is added as a subtree in the repo. This is technically a breaking change. The change was actually submitted by the python team and was the only team using at the moment. We were able to pick up this fix this because of the nature of the change and limited exposure. I consulted with fabio before pulling in this change.